src/gluonts/torch/modules/piecewise_linear.py

```diff
@@ -0,0 +1,264 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Dict, Optional, Tuple, List, cast
+
+import torch
+import torch.nn.functional as F
+
+from gluonts.core.component import validated
+from gluonts.torch.modules.distribution_output import (
+    Distribution,
+    DistributionOutput,
+)
+
+from torch.distributions import (
+    AffineTransform,
+    TransformedDistribution,
+)
+
+
+class PiecewiseLinear(Distribution):
```

Comment:
Analogously to the structure of `torch`, that keeps distributions classes within `torch.distributions`, I would put this whole module into `gluonts.torch.distributions`